From 8c40981a5f04a34d593a748b61adfd05d0cb9cac Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Wed, 15 Feb 2017 01:45:56 +0900
Subject: [PATCH 2400/2848] HADOOP-14058. Fix
 NativeS3FileSystemContractBaseTest#testDirWithDifferentMarkersWorks.
 Contributed by Yiqun Lin.

(cherry picked from commit b9f8491252f5a23a91a1d695d748556a0fd803ae)
(cherry picked from commit 25ac54a52eff0d5b8f4bee83b6f4471277a3ac61)

Change-Id: I0fa12c4a752065d13cefdb1584a5d498705ec054
---
 .../NativeS3FileSystemContractBaseTest.java        |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3native/NativeS3FileSystemContractBaseTest.java b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3native/NativeS3FileSystemContractBaseTest.java
index 3b50515..d303a92 100644
--- a/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3native/NativeS3FileSystemContractBaseTest.java
+++ b/hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3native/NativeS3FileSystemContractBaseTest.java
@@ -85,7 +85,7 @@ private void createTestFiles(String base) throws IOException {
 
   public void testDirWithDifferentMarkersWorks() throws Exception {
 
-    for (int i = 0; i < 3; i++) {
+    for (int i = 0; i <= 3; i++) {
       String base = "test/hadoop" + i;
       Path path = path("/" + base);
 
-- 
1.7.9.5

